soap/src/java/com/zimbra/soap/type/SearchSortBy.java

```diff
@@ -21,14 +21,52 @@
 
 import javax.xml.bind.annotation.XmlEnum;
 
+import com.zimbra.common.mailbox.ZimbraSortBy;
 import com.zimbra.common.service.ServiceException;
 import com.zimbra.common.zclient.ZClientException;
 
 @XmlEnum
 public enum SearchSortBy {
     // case must match protocol
-    dateDesc, dateAsc, subjDesc, subjAsc, nameDesc, nameAsc, durDesc, durAsc, none,
-    taskDueAsc, taskDueDesc, taskStatusAsc, taskStatusDesc, taskPercCompletedAsc, taskPercCompletedDesc, rcptAsc, rcptDesc;
+    dateDesc(ZimbraSortBy.dateDesc),
```

Comment:
Might want to check the before and after WSDL to make sure that it uses the correct values.  Probably does but...